platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformWiring.java

```diff
@@ -561,7 +561,7 @@ public void bind(
             @NonNull final StateSignatureCollector stateSignatureCollector,
             @NonNull final TransactionPrehandler transactionPrehandler,
             @NonNull final ConsensusRoundHandler consensusRoundHandler,
-            @NonNull final EventStreamManager defaultEventStreamManager,
+            @NonNull final EventStreamManager EventStreamManager,
```

Comment:
You accidentally capitalized the first letter of this paramter (also, doc formatting got messed up

Reply:
doh